tools/crd-validation-generator/validation-generator.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	extv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+)
+
+func main() {
+
+	dirname := flag.String("crdDir", "./manifests/generated/", "path to directory with crds from where validation field will be parsed")
```

Comment:
wouldn't `staging/src/kubevirt.io/client-go/config/crd/` be a better default at this point?

Reply:
indeed